pkg/manager/member/tikv_member_manager.go

```diff
@@ -807,6 +807,11 @@ func getTiKVStore(store *pdapi.StoreInfo) *v1alpha1.TiKVStore {
 }
 
 func (m *tikvMemberManager) setStoreLabelsForTiKV(tc *v1alpha1.TidbCluster) (int, error) {
+	if m.deps.NodeLister == nil {
```

Comment:
What about checking the lister in the caller function https://github.com/pingcap/tidb-operator/blob/master/pkg/manager/member/tikv_member_manager.go#L213?